python_modules/dagster/dagster/_daemon/auto_run_reexecution/auto_run_reexecution.py

```diff
@@ -176,6 +177,7 @@ def retry_run(
     )
 
     instance.submit_run(new_run.run_id, workspace)
+    instance.add_run_tags(failed_run.run_id, {AUTO_RETRY_RUN_ID_TAG: new_run.run_id})
```

Comment:
just confirming that this is also being set in internal

Reply:
yes - here's where we call the oss retry_run in internal https://github.com/dagster-io/internal/blob/2b08e9f07bba9660520fdb12c3cb7c5c503ff975/dagster-cloud/python_modules/dagster-cloud-backend/dagster_cloud_backend/daemon/event_log_consumer/updated_run_consumer.py#L64